src/commons/s3.rs

```diff
@@ -96,24 +99,6 @@ impl InlinedS3BucketSpec {
     }
 }
 
-/// The implementation the product should use when interacting with S3
-/// It is sometimes called protocol as it is prepended to the connection endpoint, e.g. `s3a://<host>`
-/// but it often specifies which s3 client implemtation to use, the used protocol on the wire is always S3.
-/// Spark e.g. supports `s3a` and `cos` as S3 connectors/implementations, which are different implementations but talk to the same S3 endpoint.
-pub enum S3ConnectionImplementation {
```

Comment:
This seems to mostly have been a Hadoopism anyway, I assume that you've checked that that still works?

Reply:
Yes, testing with the spark-k8s-operator.